src/test/specific_tests.py

```diff
@@ -756,5 +757,40 @@ def test_mi_term_has_corrected_url():
             "and o.url = 'http://dip.doe-mbi.ucla.edu/'" \
             "return count(o) as counter"
     result = execute_transaction(query)
+    for record in result:
+        assert record["counter"] > 0
+
+
+def test_rgd_dej_has_rgd_full_url_cross_reference():
```

Comment:
It will be a little easier to merge back to the cleanup pull request if you use capitals for the words like MATCH in the query and surround the full query with triple quotes instead of single quotes for each line. This makes it easier to cut and past the full query. Other than this minor comment I approve.

Reply:
I changed just the new tests to be triple quoted with capital words for reserved cypher words.